Jenkinsfile

```diff
@@ -500,7 +500,7 @@ try {
           init_git()
           unpack_lib('cpu')
           timeout(time: max_time, unit: 'MINUTES') {
-              sh "${docker_run} cpu ./perl-package/test.sh"
+            sh "${docker_run} cpu ./perl-package/test.sh"
```

Comment:
Remove whitespace change

Reply:
It's quite common practice to fix whitespace issues on small commits.  Can you elaborate on why you think it should be removed?

Reply:
In projects I've worked on, it was common practice not to commit whitespace changes if they are not in direct relation to the code one has modified. Reason being that this might increase the number of merge conflicts (in general) and especially messes with the history if you try to use git blame. I personally like to have a clear change history instead of commits doing something different and fixing identing on the fly.

Reply:
@marcoabreu totally agree, and I would like to see more hygienic commits in general. Not sure this case is worth an additional PR to fix space given also the cost of running all the tests etc. Up to you two. There's always good exceptions to rules right :-)

Reply:
I'd just leave the indenting as-is, considering that this is not severe. We can modify it the next time that part is touched.